c2corg_api/views/area.py

```diff
@@ -32,7 +32,7 @@ def get(self):
             permission='moderator')
     def collection_post(self):
         return self._collection_post(
-            Area, schema_area, after_add=insert_associations)
```

Comment:
Not a fix, a small clean-up. The class was not used in `_collection_post`, that's why I removed it.